include/CL/cl_exp_defined_builtin_kernels.h

```diff
@@ -368,4 +369,34 @@ typedef struct
   cl_dbk_properties_exp kernel_props[CL_DBK_MAX_PROPERTIES_EXP];
 } cl_dbk_attributes_convert_exp;
 
+/**
+ * name: "ggml_set_rows_exp"
```

Comment:
Can we name it in a more generic fashion and not refer to a ML framework/format in the DBK name? Don't other frameworks have a similar operation?

Reply:
The closest operation other frameworks have (scatter-nd in ONNX, TF and OpenVINO, index_put in Pytorch) are more general than the operation here and lacks the special broadcasting of the indices (the `indices[b0 % I0, b1 % I1, r]` part), so I'm hesitant to rename the operation to something more generic.

Reply:
Does it make sense as a DBK (likely HW accelerated) or should it be implemented via software kernels?

Reply:
But does it support this operation in hardware?

Reply:
OK. Since this is a generic operation as such, I think we can just drop 'ggml' from the name?